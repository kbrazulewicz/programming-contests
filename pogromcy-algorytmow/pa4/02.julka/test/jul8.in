999999999999999999999999999999999888888888888888888888888889999999999999999999888888888888888889999
19
