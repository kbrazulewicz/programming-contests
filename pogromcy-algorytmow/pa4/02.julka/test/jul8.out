499999999999999999999999999999999944444444444444444444444444999999999999999999944444444444444445009
499999999999999999999999999999999944444444444444444444444444999999999999999999944444444444444444990
